src/redux/actions/projectActions.test.ts

```diff
@@ -76,14 +76,41 @@ describe("Project Redux Actions", () => {
         projectServiceMock.prototype.save = jest.fn((project) => Promise.resolve(project));
 
         const project = MockFactory.createTestProject("TestProject");
-        const projectToken = appSettings.securityTokens
-            .find((securityToken) => securityToken.name === project.securityToken);
-
         const result = await projectActions.saveProject(project)(store.dispatch, store.getState);
 
         expect(result.version).toEqual(appInfo.version);
     });
 
+    it("Save Project action on new project correctly add default export format", async () => {
+        projectServiceMock.prototype.save = jest.fn((project) => Promise.resolve(project));
+
+        const skeletonProject = MockFactory.createTestProject("TestProject");
```

Comment:
nit: I would call this either "project" or "testProject" like we have been doing in the rest of the tests

Reply:
project is below, this one is use to create the test project